include/nvtop/extract_gpuinfo_common.h

```diff
@@ -184,9 +197,11 @@ struct gpu_vendor {
 
   void (*populate_static_info)(struct gpu_info *gpu_info);
   void (*refresh_dynamic_info)(struct gpu_info *gpu_info);
+  void (*refresh_utilisation_rate)(struct gpu_info *gpu_info);
 
   void (*refresh_running_processes)(struct gpu_info *gpu_info);
   char *name;
+  /* void *priv; */
```

Comment:
Do you plan on using this or can the comment be removed?

Reply:
Sorry about this, it was going to implement some sort of backend-specific private data flow and forgot to delete it before the PR.